test/cts/tool/CTSConverter/src/cts_generator.py

```diff
@@ -349,6 +351,19 @@ def DumpJSTest(model, example, js_fd):
             if t.type == "FLOAT16":
                 t.type = "FLOAT32"
 
+    '''
+    # select specifying type models
+    select_specifying_flag = False
+    if model.operations[0].optype == "CONV_2D" or model.operations[0].optype == "DEPTHWISE_CONV_2D":
+        if model.operands[0].type.type == "TENSOR_QUANT8_ASYMM_SIGNED" and \
+           model.operands[1].type.type == "TENSOR_QUANT8_SYMM_PER_CHANNEL":
+            select_specifying_flag = True
+
+    if select_specifying_flag == False:
```

Comment:
@cuiyanx Don't compare boolean values to True or False using ==, please update this code
```python
if select_specifying_flag == False:
```
to
```python
if not select_specifying_flag:
```